packages/lib/src/components/Pix/types.ts

```diff
@@ -0,0 +1,23 @@
+import {QRLoaderContainerProps} from "../helpers/QRLoaderContainer";
```

Comment:
```suggestion
import { QRLoaderContainerProps } from '../helpers/QRLoaderContainer';
```

Reply:
Fixed all files, it was prettier not running